assets/src/application/ui/calendars/dateDisplay/MediumCalendarDate/style.css

```diff
@@ -4,20 +4,12 @@
 	font-weight: normal;
 	line-height: var(--ee-font-size-default);
 	height: 100%;
+	min-height: calc(var(--ee-padding-micro) * 45);
+	min-width: calc(var(--ee-padding-micro) * 34);
```

Comment:
magic numbers :smiley:

Reply:
do you mean the 34 and 45 ?
yeah they are kind of arbitrarily set...
there's no corresponding source for those values other than that's what I felt looked best